setup/MonsterConfig.m

```diff
@@ -65,7 +65,7 @@
 			Logs.dateFormat = 'yyyy-mm-dd_HH.MM.SS';
 			Logs.logLevel = 'NFO';
 			Logs.logPath = 'logs/';
-			Logs.logFile = strcat(Logs.logPath, datestr(datetime, Logs.dateFormat));
+			Logs.logFile = strcat(Logs.logPath, datestr(datetime, Logs.dateFormat), '.log');
```

Comment:
adding '.log' extension to logs name make it easier to open, process or even ignore it from being added to git